apps/status-app/src/app/pages/status.tsx

```diff
@@ -86,52 +84,59 @@ const ServiceStatusPage = (): JSX.Element => {
   const Services = () => {
     return (
       <div className="small-container">
-        <PageLoader />
-        <Title data-testid="service-name">All {capitalizeFirstLetter(tenantName)} services</Title>
-        <div className="section-vs">
-          These are the services currently being offered by{' '}
-          {location.pathname.slice(1) ? capitalizeFirstLetter(tenantName) : 'the Alberta Digital Service Platform'}. All
```

Comment:
Minor thing.
"the Alberta Digital Service Platform"

Should "the" be capitalize to "The" ?

Other than that everything looks good.

Reply:
That is part of paragraph, it is fine for lower case, thanks for reviewing.